modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -610,11 +611,19 @@ public Map<String, Object> getSystemModelAttributes(
 				_objectFieldLocalService.getObjectFields(
 					objectDefinition.getObjectDefinitionId())) {
 
-			modelAttributes.put(
-				objectField.getName(),
-				GetterUtil.getObject(
-					baseModelAttributes.get(objectField.getDBColumnName()),
-					objectEntryId));
+			Object value = GetterUtil.getObject(
+				baseModelAttributes.get(objectField.getDBColumnName()),
+				objectEntryId);
+
+			if (Objects.equals(
+					objectField.getDBType(),
+					ObjectFieldConstants.DB_TYPE_STRING)) {
+
+				value = LocalizationUtil.getLocalization(
+					(String)value, null, true);
```

Comment:
to make code cleaner, should reference to the below code-lines
if( value instanceof String ) {
	value = LocalizationUtil.getLocalization((String)value, null, true);
}

Reply:
I fixed follow this comment @thienquach87